### DISCUSSION TITLE
dos installer scripts finally integrated

### DISCUSSION THREAD

**Discussion by stsp** [dosemu2/dosemu2#1503] [General]
Just wanted to let you know that
I finally packaged the install-otherdos
scripts, and wrote a wiki page about
them:
https://github.com/dosemu2/dosemu2/wiki/installing-various-DOSes

I am sure @mateuszviste @mceric and
@dufresnep are going to be interested
in that.

**Comment by jschwartzenberg** [comment]
Great! I hope the choice for Python for these scripts will also work out. Originally I started writing them in Bash. Everything mostly worked but then I felt that Python seemed a better fit and ported them over. I like the direction @stsp is taking with further improvements.

I also wrote some Python code that can integrate device drivers into a Windows 3.1 installer and trigger the installation:
https://github.com/dosemu2/install-win31

Maybe more tooling can be automatically installed and set up optimally for DOSEMU this way. Having a defacto library in the form of the Internet Archive (archive.org) also helps there.

**Reply by stsp** [reply]
> I like the direction @stsp is taking with further improvements.

Thanks. :)
Note that this is the first time
I am trying python. Very harsh
feeling when you can read and
mostly understand what is written,
but can't type a single line yourself. :)
So had to learn a bit.

> I also wrote some Python code that can integrate device drivers into a Windows 3.1 installer and trigger the installation:

I believe this can better be done
with bat files. Note that now we have
curl/wget on the DOS side.

> Maybe more tooling can be automatically installed and set up optimally for DOSEMU this way.

Perhaps you can resurrect your
DOS installers that I deleted, as
yet another package. But this time,
they won't install them to dosemu
directly, but just pack. Then we
will hook their produced archives
to install-otherdos.
